Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.h

```diff
@@ -185,6 +186,7 @@ class RemoteRenderingBackendProxy
 
     RenderingUpdateID m_renderingUpdateID;
     RenderingUpdateID m_didRenderingUpdateID;
+    bool m_shouldBlockIOSurfaceCreationForNextImageBuffer;
```

Comment:
This is missing a default initializer so it is undefined what its initial value will be.

Reply:
Fixed.